packages/jss/src/renderers/DomRenderer.js

```diff
@@ -214,7 +212,7 @@ function findPrevNode(options: PriorityOptions): PrevNode | false {
 
     // If user specifies an insertion point and it can't be found in the document -
     // bad specificity issues may appear.
-    warning(insertionPoint === 'jss', '[JSS] Insertion point "%s" not found.', insertionPoint)
+    warning(false, `[JSS] Insertion point "${insertionPoint}" not found.`)
```

Comment:
Not sure why we are checking for `jss` here, when the user wants a custom insertion point, it should exist.

Reply:
I think it was there because we used to have a default insertion point 'jss', which should not warn if its not found.